modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase

```diff
@@ -217,12 +299,22 @@ definition {
 	}
 
 	@description = "COMMERCE-5864. As a buyer, I want to view the product price"
-	@ignore = "true"
 	@priority = "3"
 	test ViewProductCardPrice {
 		property portal.acceptance = "false";
+		property test.name.skip.portal.instance = "CommerceProductCard#ViewProductCardPrice";
+
+		CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+		Navigator.openWithAppendToBaseURL(urlAppend = "web/minium");
 
-		// TODO COMMERCE-5864 ViewProductCardPrice pending implementation
+		Click(
+			key_productName = "U-Joint",
+			locator1 = "CommerceAccelerators#PRODUCT_CARD_NAME");
+
+		AssertTextEquals.assertPartialText(
+			locator1 = "CommerceAccelerators#PRODUCT_DETAILS_LIST_PRICE",
+			value1 = "24.00");
```

Comment:
@nikki-pru As per the ticket's description, this test must assert that the currency is correct in the product card, so the test is not supposed to click the product and enter its details.

Reply:
In progress

Reply:
Done